### ISSUE TITLE
[BUG] Mailer functionality is not work

### ISSUE THREAD

**Issue by abicur** [RocketChat/Rocket.Chat#12745]
### Description:

A Mailer in administative panel is not work.

### Steps to reproduce:

1. Go to admin panel and then to the Mailer.
2. Fill form fields.
3. Click "Send Email" button.

### Expected behavior:

Emails must be send.

### Actual behavior:

Error popup - "Internal server error"

### Server Setup Information:

- Version of Rocket.Chat Server: 0.71.1
- Operating System: ubuntu 
- Deployment Method: snap
- Number of Running Instances: 1
- DB Replicaset Oplog: 1
- NodeJS Version: v8.11.3
- MongoDB Version: 3.6.3

### Relevant logs:

`
Exception while invoking method 'Mailer.sendMail' TypeError: Mailer.sendMail is not a function     at MethodInvocation.Mailer.sendMail (/snap/rocketchat-server/1343/programs/server/packages/rocketchat_mailmessages.js:204:19)     at MethodInvocation.methodMap.(anonymous function) (packages/rocketchat_monitoring.js:2731:30)     at MethodInvocation.methodsMap.(anonymous function) (/snap/rocketchat-server/1343/programs/server/packages/rocketchat_lib.js:2518:36)     at maybeAuditArgumentChecks (/snap/rocketchat-server/1343/programs/server/packages/ddp-server.js:1877:12)     at DDP._CurrentMethodInvocation.withValue (/snap/rocketchat-server/1343/programs/server/packages/ddp-server.js:902:126)     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1186:12)     at DDPServer._CurrentWriteFence.withValue (/snap/rocketchat-server/1343/programs/server/packages/ddp-server.js:902:98)     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1186:12)     at Promise (/snap/rocketchat-server/1343/programs/server/packages/ddp-server.js:902:46)     at new Promise (<anonymous>:null:null)     at Session.method (/snap/rocketchat-server/1343/programs/server/packages/ddp-server.js:875:23)     at /snap/rocketchat-server/1343/programs/server/packages/ddp-server.js:754:85  
`

**Comment by MarcosSpessatto** [comment]
@abicur can you please update your version of rocket.chat server? In `v0.72.0` this issue has been fixed. I'm closing this issue, if the problem persists, feel free to reopen. Thanks.

**Comment by ggazzo** [comment]
thanks @MarcosSpessatto :)

**Comment by SteveUNSW** [comment]
Still not working:

[34mI20181130-00:23:12.660(0) Exception while invoking method 'verifyEmail' Error: Can't call setUserId in a method after calling unblock     at MethodInvocation.setUserId (/app/bundle/programs/server/packages/ddp-common.js:367:13)     at AccountsServer._loginUser (/app/bundle/programs/server/packages/accounts-base.js:700:22)     at AccountsServer._attemptLogin (/app/bundle/programs/server/packages/accounts-base.js:744:56)     at AccountsServer._loginMethod (/app/bundle/programs/server/packages/accounts-base.js:762:17)     at MethodInvocation.verifyEmail (/app/bundle/programs/server/packages/accounts-password.js:1040:21)     at MethodInvocation.methodsMap.(anonymous function) (/app/bundle/programs/server/packages/rocketchat_lib.js:2696:36)     at MethodInvocation.methodMap.(anonymous function) (packages/rocketchat_monitoring.js:2731:30)     at maybeAuditArgumentChecks (/app/bundle/programs/server/packages/ddp-server.js:1877:12)     at DDP._CurrentMethodInvocation.withValue (/app/bundle/programs/server/packages/ddp-server.js:902:126)     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1304:12)     at DDPServer._CurrentWriteFence.withValue (/app/bundle/programs/server/packages/ddp-server.js:902:98)     at Meteor.EnvironmentVariable.EVp.withValue (packages/meteor.js:1304:12)     at Promise (/app/bundle/programs/server/packages/ddp-server.js:902:46)     at new Promise (<anonymous>:null:null)     at Session.method (/app/bundle/programs/server/packages/ddp-server.js:875:23)     at /app/bundle/programs/server/packages/ddp-server.js:754:85

**Comment by MarcosSpessatto** [comment]
Closing in favor of: https://github.com/RocketChat/Rocket.Chat/issues/12762
